org.adempiere.base/src/org/compiere/model/MAttachmentEntry.java

```diff
@@ -108,6 +108,9 @@ public MAttachmentEntry(MAttachmentEntry copy)
 	/** Lazy Data Source */
 	private IAttachmentLazyDataSource m_ds = null;
 
+	/** True if the entry has been updated (on upload, when a file with same name is already there) */
```

Comment:
"(on upload, when a file with same name is already there)" - this part doesn't seems appropriate here (exact behaviour is storage provider dependent).

Reply:
javadoc updated